packages/core/src/components/GlobalActions/GlobalActions.stories.tsx

```diff
@@ -183,3 +192,73 @@ export const SectionVariant: StoryObj<HvGlobalActionsProps> = {
     );
   },
 };
+
+export const CustomContent: StoryObj<HvGlobalActionsProps> = {
+  argTypes: {
+    classes: { control: { disable: true } },
+  },
+  parameters: {
+    docs: {
+      description: {
+        story:
+          "The content of the Global Actions can be customized. This is useful when you want to add different elements other than the title and actions.",
+      },
+    },
+  },
+  render: () => {
+    const classes = {
+      tab: css({ height: 42, ...theme.typography.title4 }),
+      wrapper: css({
+        display: "flex",
+        justifyContent: "space-between",
+        padding: theme.spacing(0, "xs"),
+        height: 48,
+      }),
+      actions: css({ marginLeft: "unset" }),
+      backButton: css({
+        display: "flex",
+        alignItems: "center",
+        gap: theme.space.xs,
+      }),
+    };
+
+    const [value, setValue] = useState(0);
+
+    const customTitle = (
+      <HvTabs value={value} onChange={(_, newValue) => setValue(newValue)}>
+        <HvTab className={classes.tab} label="Tab 1" />
+        <HvTab className={classes.tab} label="Tab 2" />
+        <HvTab className={classes.tab} label="Tab 3" />
+      </HvTabs>
+    );
+
+    return (
+      <HvGlobalActions
+        title={customTitle}
+        classes={{
+          wrapper: classes.wrapper,
+          actions: classes.actions,
+        }}
+        backButton={
```

Comment:
Should we deprecate the `backButton` (with the intention of removing it in v6), like we briefly mentioned, and create a new one more generic since not only a back button can be used here?